src/use-cases/authenticate.ts

```diff
@@ -42,15 +40,6 @@ export const Authenticate = (config: Config, userService: UserRepository, eventB
 
         const encodedPayload = encode(config.authentication_jwt_secret, payload, '30m');
 
```

Comment:
Should anything at all be logged here to replace the removal of auditing?

Reply:
Very good point - the intention is that the auditing will be re-implemented in submission - but I like your thinking- we should do it here as well :)

Reply:
@Cooryd added